### ISSUE TITLE
C# Web API cannot connect to the server

### ISSUE THREAD

**Issue by Sauedy** [microsoft/vscode-dotnettools#718]
Type: <b>Bug</b>

Debugging/Running a Web API results in the browser returning Browser cannot connect to the server. 

This only happens when using the prerelease version of C# Dev Kit

I am using a standard SPA react app that was created in visual studio in the past. 

Extension version: 1.1.9
VS Code version: Code 1.83.0 (Universal) (e7e037083ff4455cf320e344325dacb480062c3c, 2023-10-03T16:13:15.449Z)
OS version: Darwin arm64 23.0.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Max (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|3, 3, 4|
|Memory (System)|32.00GB (3.77GB free)|
|Process Argv|--crash-reporter-id 29c35da4-c314-460b-ae30-4cd6eb147953|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263cf:30880073
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
f6dab269:30613381
a9j8j154:30646983
showlangstatbar:30737416
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30869155
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixtcf:30871695
pythonregdiag2:30871582
pyreplss2:30886141
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
dsvsc016:30886110
dsvsc017:30886112
dsvsc018:30886114
aa_t_chat:30882232

```

</details>

**Comment by WardenGnaw** [comment]
Thank you for reporting this issue. There was a regression in v1.1.5 and now in v1.1.9. This should resolved in the next release.

Rolling back to  v1.0.12 will work around this issue for now.

**Comment by WardenGnaw** [comment]
This is fixed in v1.2.2
